betterBuilding/src/main/resources/betterbuilding.mixins.json

```diff
@@ -3,8 +3,8 @@
   "minVersion": "0.8",
   "package": "legobrosbuild.betterbuilding.mixin",
   "compatibilityLevel": "JAVA_17",
-  "mixins": [],
-  "client": ["BindingMixin","OptionScreenMixin","HudMixin"],
+  "mixins": ["BindingMixin"],
```

Comment:
you know mixins can run on the server, right?

Reply:
No I didn't